packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -5868,4 +5882,53 @@ export default class Meeting extends StatelessWebexPlugin {
       failure: `${LOG_HEADER} disable bnr failure, `
     });
   }
+
+  /**
+   * starts keepAlives being sent
+   * @returns {void}
+   * @private
+   * @memberof Meeting
+   */
+  startKeepAlive = () => {
+    if (this.keepAliveTimerId) {
+      LoggerProxy.logger.error('Meeting:index#startKeepAlive --> keepAliveTimerId already exists');
+
+      return;
+    }
+    if (!this.joinedWith?.keepAliveUrl) {
+      LoggerProxy.logger.error('Meeting:index#startKeepAlive --> no keepAliveUrl');
+
+      return;
+    }
+    if (!this.joinedWith?.keepAliveSecs) {
+      LoggerProxy.logger.error('Meeting:index#startKeepAlive --> no keepAliveSecs');
+
+      return;
+    }
+    if (this.joinedWith.keepAliveSecs <= 1) {
+      LoggerProxy.logger.error('Meeting:index#startKeepAlive --> keepAliveSecs <= 1');
+
+      return;
+    }
+    const {keepAliveUrl} = this.joinedWith;
+    const keepAliveInterval = (this.joinedWith.keepAliveSecs - 1) * 750;
```

Comment:
maybe worth adding a comment that it's copied from UCF, because it's quite weird....

Reply:
it is quite weird but not exactly wrong, so copying seemed the safe option, I will add a comment

Reply:
done